### DISCUSSION TITLE
Cooking time?

### DISCUSSION THREAD

**Discussion by tmllull** [TandoorRecipes/recipes#1790] [General]
Hi! I've a question about time fields in recipes. There are currently 2 types of time: preparation and waiting.

Preparation time it's clear for me, but I don't know what is the purpose for the waiting time… I use this field as cooking time, because I think it's a good point to know how long you have to cook a recipe or step. Or the waiting time it's for this purpose?

Adding to this, maybe will be a good idea to add hours on time fields (or convert to hours if it needs more than 60 min). For recipes that need to be cooked for a long time, such as recipes with Crockpot, for example I've some that must be cooked about 4, 6 or 8 hours, and I have to put 240, 360 or 480 min. But this it's an extra just to improve this wonderful project. ;)

Thanks for your help!

Best regards.

**Comment by smilerz** [comment]
any non-active time can be used here.  Cook time is fine, if there is a brining/marinade step or rest time after cook it could be included as well.

**Comment by Silther** [comment]
any reason why there are not 3 types of time?
preparation (cutting vegetables ...), waiting (resting the dough), cooking time (time in the oven or boiling in water)

**Comment by Draky50110** [comment]
"cooking time" maybe should be another area/text because as said above, preparation time is ok, "waiting" after cooking (or before).
Also, on imported recipe, on websites, there is always a "cooking" explicit time given.
